examples/trials/mnist-pbt-tuner-pytorch/config.yml

```diff
@@ -0,0 +1,21 @@
+authorName: default
+experimentName: example_mnist_pbt_tuner_pytorch
+trialConcurrency: 3
+maxExecDuration: 2h
+maxTrialNum: 100
+#choice: local, remote, pai
+trainingServicePlatform: local
+searchSpacePath: search_space.json
+#choice: true, false
+useAnnotation: false
+tuner:
+  codeDir: ~/nni/src/sdk/pynni/nni/pbt_tuner
```

Comment:
this tuner should be a built-in tuner, not a customized tuner.

Reply:
solved